test/e2e/components/vl-data-table.js

```diff
@@ -1,9 +1,98 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('selenium-webdriver');
 
 class VlDataTable extends VlElement {  
-    constructor(driver, selector) {
-        super(driver, selector);
+
+    async getCaption() {
+        const caption = await this.findElement(By.css('caption'));
+        return caption.getText();
+    }
+
+    async getDataTableHeader() {
+        return new VlDataTableHeader(this.driver, await this.findElement(By.css('thead')));
+    }
+
+    async getDataTableBody() {
+        return new VlDataTableBody(this.driver, await this.findElement(By.css('tbody')));
+    }
+
+    async isHover() {
+        return this.hasAttribute('hover');
+    }
+
+    async isMatrix() {
+        return this.hasAttribute('matrix');
+    }
+
+    async isLined() {
+        return this.hasAttribute('lined');
+    }
+
+    async isZebra() {
+        return this.hasAttribute('zebra');
+    }
+}
+
+class VlDataTableHeader extends VlElement {
+    async getRow() {
+        return new VlDataTableRow(this.driver, await this.findElement(By.css('tr')));
     }
 }
 
+class VlDataTableBody extends VlElement {
+    async getRows() {
+        const rows = await this.findElements(By.css('tr'));
+        
+        return Promise.all(
+            rows.map(async row => {
+                return await new VlDataTableRow(this.driver, row);
+            })
+        );
```

Comment:
De **async ... await** is denk ik niet nodig:
```suggestion
        return Promise.all(rows.map(row => new VlDataTableRow(this.driver, row)));
```

Reply:
ok